Jenkinsfile.s390x.its

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+pipeline {
+  agent { node { 's390x' } }
+  options {
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '5'))
+    timeout(time: 180, unit: 'MINUTES')
+  }
+  parameters {
+    string( defaultValue: 'master', description: 'Core Its branch (default master)',
+            name: 'ITS_BRANCH' )
```

Comment:
What's the actual value for this parameter ? It should be computed, but I don't really see any computation in this file.

Reply:
I did not imply to go through GitHub actions, but maybe the shell script from the GitHub action could be reused somehow in the Jenkins pipeline in order to find the correct remote repository + branch to use.

Reply:
I see the existing Jenkinsfile.its was added by @olamy.   @olamy can you please help answer this - how does the ITS_BRANCH variable get computed?  And how is integration test triggered for Jenkins builds?  Thanks.

Reply:
As far as I can see with this commit https://github.com/apache/maven/commit/bcfea094c00c7064bb654717599bc1c7fc842ecd the file has been by @kun-lu20

Reply:
I'm referring to this commit (https://github.com/apache/maven/commit/b3e3972988df7b3dde26ccd4e48b3e6b97d89ad6, PR: https://github.com/apache/maven/pull/355 ) that added the original [Jenkinsfile.its](https://github.com/apache/maven/blob/master/Jenkinsfile.its).